packages/flutter_test/pubspec.yaml

```diff
@@ -1,10 +1,18 @@
 name: flutter_test
 dependencies:
-  test: 0.12.13
   quiver: ^0.21.4
 
-  # Not needed directly but transitively included due to package:test.
+  test: 0.12.13
   analyzer: 0.27.2
 
+  # The flutter tools depend on very specific internal implementation
```

Comment:
for consistency with the other three notes, you could add "A note about 'test'" here.

Or, put the comments immediately below the lines that declare the versions.

Or, do whatever you want. I feel like I'm nit picking trivial bike shed colours here.